TeamSnapSDK/SDK/DataTypes/TSDKEvent.h

```diff
@@ -6,6 +6,20 @@
 #import "TSDKCollectionObject.h"
 #import "TSDKObjectsRequest.h"
 
+typedef enum TSDKRepeatingEventTypeCode : NSInteger  {
```

Comment:
Use NS_ENUM() macro for better interoperability with Swift. https://developer.apple.com/library/content/documentation/Swift/Conceptual/BuildingCocoaApps/InteractingWithCAPIs.html#//apple_ref/doc/uid/TP40014216-CH8-ID21